app/app.js

```diff
@@ -0,0 +1,8 @@
+var app = angular.module('MyApp', [
```

Comment:
eliminate `var app =`

Reply:
rename `app.js` to `my-app.module.js` and module to 'my-app.module'

Reply:
done